macOS/LocalPackages/FeatureFlags/Sources/FeatureFlags/FeatureFlag.swift

```diff
@@ -56,19 +56,28 @@ public enum FeatureFlag: String, CaseIterable {
     case autcompleteTabs
     case webExtensions
     case syncSeamlessAccountSwitching
+    case popoverVsBannerExperiment
```

Comment:
The name `popoverVsBannerExperiment` isn’t very informative. We can keep it like this, but could you add the Asana task URL in the comment?

Reply:
Great callout. Done